src/main.cpp

```diff
@@ -638,22 +638,110 @@ unsigned int LimitOrphanTxSize(unsigned int nMaxOrphans) EXCLUSIVE_LOCKS_REQUIRE
     return nEvicted;
 }
 
-bool IsFinalTx(const CTransaction &tx, int nBlockHeight, int64_t nBlockTime)
+int64_t LockTime(const CTransaction &tx, int flags, const CCoinsView* pCoinsView, int nBlockHeight, int64_t nBlockTime)
```

Comment:
No it switches to opposite meaning -- zero indicates tx is final, nonzero
indicates when it will become final.
On Aug 18, 2015 8:24 PM, "Luke-Jr" <notifications@github.com> wrote:

> In src/main.cpp
> <https://github.com/bitcoin/bitcoin/pull/6312#discussion_r37376960>:
>
> > @@ -638,22 +638,110 @@ unsigned int LimitOrphanTxSize(unsigned int nMaxOrphans) EXCLUSIVE_LOCKS_REQUIRE
> >      return nEvicted;
> >  }
> >
> > -bool IsFinalTx(const CTransaction &tx, int nBlockHeight, int64_t nBlockTime)
> > +int64_t LockTime(const CTransaction &tx, int flags, const CCoinsView* pCoinsView, int nBlockHeight, int64_t nBlockTime)
>
> You mean because it returns int64_t instead of bool? But unless I missed a
> spot, the return value is only ever *used* as a bool in the same sense...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitcoin/bitcoin/pull/6312/files#r37376960>.
>